geomstats/backend/numpy.py

```diff
@@ -2,8 +2,6 @@
 
 import numpy as np
 
-pi = np.pi
-
 
 def array(val):
```

Comment:
should we order the functions by alphabetic order? it would be easier to have a look through it later, or to add new functions.

Reply:
yes absolutely - right now I'm adding them at the end but I'll do this refactoring (or find a way to automate it) later